### PR TITLE
Address Dart 2 infos

### PR DESCRIPTION
**Pull request by jacehensley-wf**
## Ultimate Problem
Dart 2 dev just got released, and when pulling it into OverReact there are some new infos.

## Solution
- Address all infos

## Testing Suggestions
- Run
```bash
brew upgrade dart --devel --with-content-shell --with-dartium
```
- Verify you have Dart version `2.0.0-dev.1.0`
- Verify that there are no infos

## Possible Areas of Regression
- N/A

---
FYA: @Workiva/ui-platform-pp

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: lib/src/util/map_util.dart [hunk 1]
```diff
@@ -56,13 +56,13 @@ Map getPropsToForward(Map props, {
   }
 
   if (onlyCopyDomProps) {
-    new List.from(propsToForward.keys).forEach((String key) {
-      if (key.startsWith('aria-')) return;
-      if (key.startsWith('data-')) return;
-      if (_validDomProps.contains(key)) return;
+    for (var key in new List.from(propsToForward.keys)) {
```
Comment (aaronlademann-wf) [inline lib/src/util/map_util.dart]: The last time we changed a `forEach` to a `for` loop... we broke a bunch of stuff... can we verify that we didn't do that again here? _(I don't recall specifically what went wrong last time)_

Comment (johnbland-wf) [reply lib/src/util/map_util.dart]: I believe it was this same sort of loop too where you had the list manipulation within the declaration.

@clairesarsam-wf found/fixed it so she might recall the specifics.

Comment (greglittlefield-wf) [reply lib/src/util/map_util.dart]: It was due to `return` not being swapped out with `continue`

Comment (johnbland-wf) [reply lib/src/util/map_util.dart]: That's right.

**Pull request closed without merging**